### ISSUE TITLE
options.allowedHosts[0] should be a non-empty string.

### ISSUE THREAD

**Issue by YasineNifa** [peanutsee/reactjs-django-boilerplate#1]
Hello guys!
I have this error when I try to start the app ` npm start `
`
Invalid options object. Dev Server has been initialized using an options object that does not match the API schema. options.allowedHosts[0] should be a non-empty string.
`
The backend is successfully running.
How can I fix that ?

**Comment by peanutsee** [comment]
Did you check the proxy settings in your package.json?

**Comment by YasineNifa** [comment]
Yes, I have this `"proxy": "http://127.0.0.1:8000"` in my package.json file

**Comment by peanutsee** [comment]
Ok, and what is your environment - node version please?
